.travis.yml

```diff
@@ -10,6 +10,4 @@ before_script:
   - pip3 install pre-commit
 script:
   - pre-commit run -a
-# To uncomment when we have meaningful code
-# - npm install --log-level=error
-# - npm run lint
+  - npm run lint-all
```

Comment:
I think its a better idea to only lint/test/build the stuff that was changed in a given pr. Since Nx offered the option I thought it was pretty cool.